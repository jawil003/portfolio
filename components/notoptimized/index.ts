export { NotOptimized } from "./notoptimized";
export type { Props as NotOptimizedProps } from "./notoptimized";
