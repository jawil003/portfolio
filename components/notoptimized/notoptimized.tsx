import { css } from "@emotion/react";
import { MobileApplicationDesign } from "../../designs/mobileApplication.design";
import React from "react";
import { FlexContainer } from "../shared/flexcontainer/flexcontainer";
import { Typography } from "../shared/typography";
import { NavigationBar } from "../shared/navigationbar/navigationbar";

export interface Props {}

/**
 * An NotOptimized React Component.
 * @author Jannik Will
 * @version 0.1
 */
export const NotOptimized: React.FC = () => {
  return (
    <>
      <NavigationBar />
      <FlexContainer
        direction="column"
        justifyContent="center"
        alignItems="center"
        rowGap="50px"
        css={css`
          & {
            margin: 0px;
            padding: 50px;
            width: 100%;
            height: 100%;
          }
        `}
      >
        <MobileApplicationDesign width="100%" />
        <Typography
          css={css`
            & {
              font-size: 20px;
            }
          `}
          align="center"
          variant="b1"
          bold="semi-bold"
        >
          Diese Seite ist zurzeit noch
          nicht vollständig für mobile
          Geräte optimiert
        </Typography>
      </FlexContainer>
    </>
  );
};
