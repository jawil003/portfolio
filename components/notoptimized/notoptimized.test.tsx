import { render } from "@testing-library/react"
import { NotOptimized } from "./notoptimized";

describe("NotOptimized", ()=>{
    test("Create NotOptimized", ()=>{
       const notOptimized =  render(<NotOptimized/>)

        expect(notOptimized.baseElement).toMatchSnapshot();
    })
})