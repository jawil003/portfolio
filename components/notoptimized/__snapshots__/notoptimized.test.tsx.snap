// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotOptimized Create NotOptimized 1`] = `
.emotion-0 {
  position: fixed;
  top: 0;
  z-index: 998;
  margin-top: 10px;
  height: 100px;
  padding: 15px;
}

.emotion-0>.desktop-navigation-bar-hamburger-menu-container {
  height: 100%;
}

.emotion-1 {
  display: none;
  position: fixed;
  z-index: 999;
  top: 0;
  left: 0;
  width: 100vw;
  height: 100vh;
  transform-origin: left center;
  background: rgba(
                    0,
                    0,
                    0,
                    0.35
                  );
}

.emotion-1>.desktop-navigation-bar-sidebar-container {
  position: relative;
  z-index: 1000;
  display: grid;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: start;
  justify-content: start;
  padding: 10px 0px;
  -webkit-align-content: start;
  -ms-flex-line-pack: start;
  align-content: start;
  -webkit-align-items: start;
  -webkit-box-align: start;
  -ms-flex-align: start;
  align-items: start;
  grid-template-rows: repeat(3, auto) 1fr auto;
  float: left;
  height: 100%;
  background: #f3f3f3;
}

.emotion-1 .desktop-navigation-bar-logo-container {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  padding: 0px 20px;
}

.emotion-2 {
  padding: 20px;
  cursor: pointer;
  border-radius: 10px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  position: relative;
  overflow: visible;
}

.emotion-2>.logo-beta-label {
  background-color: #00739d;
  color: #f3f3f3;
  padding: 10px;
  font-size: 1.125rem;
  font-weight: 600;
  border-radius: 34px;
  position: absolute;
  bottom: 10px;
  right: -30px;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-flex-wrap: no-wrap;
  -webkit-flex-wrap: no-wrap;
  -ms-flex-wrap: no-wrap;
  flex-wrap: no-wrap;
  margin: calc(10px / -2) calc(0px / -2);
}

.emotion-3>* {
  margin: calc(10px / 2) calc(0px / 2);
}

.emotion-4 {
  padding: 10px 20px;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #000;
  cursor: pointer;
}

.emotion-4>svg {
  fill: #000;
}

.emotion-4:hover {
  background-color: rgba(
                  0,
                  0,
                  0,
                  0.35
                );
}

.emotion-4>* {
  display: inline-block;
}

.emotion-5 {
  text-align: start;
  font-style: normal;
  margin: unset;
  -webkit-text-decoration: none;
  text-decoration: none;
  letter-spacing: normal;
  word-spacing: normal;
  color: #000;
  word-wrap: unset;
  padding: unset;
  word-break: normal;
}

.emotion-5 {
  font-weight: 600;
  font-size: 1em;
}

.emotion-7 {
  padding: 10px 20px;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #000;
  cursor: pointer;
}

.emotion-7>svg {
  fill: #000;
}

.emotion-7:hover {
  background-color: rgba(
                0,
                0,
                0,
                0.35
              );
}

.emotion-7>* {
  display: inline-block;
}

.emotion-14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-flex-wrap: no-wrap;
  -webkit-flex-wrap: no-wrap;
  -ms-flex-wrap: no-wrap;
  flex-wrap: no-wrap;
  margin: calc(50px / -2) calc(0px / -2);
}

.emotion-14>* {
  margin: calc(50px / 2) calc(0px / 2);
}

.emotion-14 {
  margin: 0px;
  padding: 50px;
  width: 100%;
  height: 100%;
}

.emotion-15 {
  text-align: center;
  font-style: normal;
  margin: unset;
  -webkit-text-decoration: none;
  text-decoration: none;
  letter-spacing: normal;
  word-spacing: normal;
  color: #000;
  word-wrap: unset;
  padding: unset;
  word-break: normal;
}

.emotion-15 {
  font-weight: 600;
  font-size: 1em;
}

.emotion-15 {
  font-size: 20px;
}

<body>
  <div>
    <nav
      class="desktop-navigation-bar-menu-container undefined emotion-0"
    >
      <div
        class="desktop-navigation-bar-hamburger-menu-container"
      >
        <svg
          class="hamburger-menu-icon undefined"
          clip-rule="evenodd"
          fill="#00739d"
          fill-rule="evenodd"
          height="100%"
          stroke-linejoin="round"
          stroke-miterlimit="2"
          viewBox="0 0 2481 2481"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
            fill="#00739d"
            transform="matrix(.7503 0 0 .7503 306.782 308.462) matrix(0 -1.1652 1.52426 0 -712.287 2973.76)"
          />
          <path
            d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
            fill="#00739d"
            transform="matrix(.7503 0 0 .7503 306.782 308.462) matrix(0 -1.1652 1.52426 0 -712.287 2076.49)"
          />
          <path
            d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
            fill="#00739d"
            transform="matrix(.7503 0 0 .7503 306.782 308.462) matrix(0 -1.1652 1.52426 0 -712.287 1179.26)"
          />
        </svg>
      </div>
    </nav>
    <nav
      class="desktop-navigation-bar-sidebar-container-background  emotion-1"
    >
      <div
        class="desktop-navigation-bar-sidebar-container"
        style="transform: translateX(-200px) translateZ(0);"
      >
        <div
          class="desktop-navigation-bar-logo-container"
        >
          <div
            class="logo-container emotion-2"
          >
            <svg
              fill-rule="evenodd"
              height="60px"
              viewBox="0 0 1925 1191"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
                fill="url(#a)"
                transform="matrix(.43892 -.2534 .3315 .57417 -346.149 44.635)"
              />
              <path
                d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
                fill="url(#b)"
                transform="matrix(.43892 -.2534 .3315 .57417 334.96 34.958)"
              />
              <path
                d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
                fill="url(#c)"
                transform="matrix(.43892 .2534 .3315 -.57417 767.637 1145.681)"
              />
              <defs>
                <lineargradient
                  gradientTransform="matrix(8.55347 1718.32 -2247.83 6.53858 701.965 411.352)"
                  gradientUnits="userSpaceOnUse"
                  id="a"
                  x1="0"
                  x2="1"
                  y1="0"
                  y2="0"
                >
                  <stop
                    offset="0"
                    stop-color="#0078a3"
                  />
                  <stop
                    offset=".27"
                    stop-color="#00749e"
                  />
                  <stop
                    offset=".5"
                    stop-color="#00678d"
                  />
                  <stop
                    offset="1"
                    stop-color="#004a65"
                  />
                </lineargradient>
                <lineargradient
                  gradientTransform="matrix(8.55347 1718.32 -2247.83 6.53858 701.965 411.352)"
                  gradientUnits="userSpaceOnUse"
                  id="b"
                  x1="0"
                  x2="1"
                  y1="0"
                  y2="0"
                >
                  <stop
                    offset="0"
                    stop-color="#0078a3"
                  />
                  <stop
                    offset=".27"
                    stop-color="#00749e"
                  />
                  <stop
                    offset=".5"
                    stop-color="#00678d"
                  />
                  <stop
                    offset="1"
                    stop-color="#004a65"
                  />
                </lineargradient>
                <lineargradient
                  gradientTransform="matrix(8.55347 1718.32 -2247.83 6.53858 701.965 411.352)"
                  gradientUnits="userSpaceOnUse"
                  id="c"
                  x1="0"
                  x2="1"
                  y1="0"
                  y2="0"
                >
                  <stop
                    offset="0"
                    stop-color="#0078a3"
                  />
                  <stop
                    offset=".27"
                    stop-color="#00749e"
                  />
                  <stop
                    offset=".5"
                    stop-color="#00678d"
                  />
                  <stop
                    offset="1"
                    stop-color="#004a65"
                  />
                </lineargradient>
              </defs>
            </svg>
          </div>
        </div>
        <div
          style="height: 40px;"
        />
        <div
          class="emotion-3"
        >
          <a
            class="emotion-4"
          >
            <div
              style="display: inline-flex; align-items: center;"
            />
            <div
              style="width: 10px;"
            />
            <span
              class="emotion-5 emotion-6"
              color="#000"
              id="subtitle1"
            >
              Willkommen
            </span>
          </a>
          <a
            aria-hidden="true"
            class="emotion-7"
            href="https://willey3x37.hashnode.dev"
            rel="noreferrer"
            tabindex="0"
            target="_blank"
          >
            <div
              style="display: inline-flex; align-items: center;"
            />
            <div
              style="width: 10px;"
            />
            <span
              class="emotion-5 emotion-6"
              color="#000"
              id="subtitle1"
            >
              Veröffentlichungen
            </span>
          </a>
          <a
            aria-hidden="true"
            class="emotion-7"
            href="https://dribbble.com/willey3x37"
            rel="noreferrer"
            tabindex="0"
            target="_blank"
          >
            <div
              style="display: inline-flex; align-items: center;"
            />
            <div
              style="width: 10px;"
            />
            <span
              class="emotion-5 emotion-6"
              color="#000"
              id="subtitle1"
            >
              Designs
            </span>
          </a>
        </div>
        <div />
        <div
          class="emotion-3"
        />
      </div>
    </nav>
    <div
      class="emotion-14"
    >
      <svg
        data-name="Layer 1"
        viewBox="0 0 831 729.989"
        width="100%"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M908.476 788.577v-72.34s28.192 51.286 0 72.34z"
          fill="#f1f1f1"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M910.218 788.564l-53.29-48.921s56.845 13.915 53.29 48.921zM290.96 788.577v-72.34s-28.193 51.286 0 72.34z"
          fill="#f1f1f1"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M289.218 788.564l53.29-48.921s-56.846 13.915-53.29 48.921z"
          fill="#f1f1f1"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M775.256 252.054h-3.863v-105.81a61.24 61.24 0 00-61.239-61.239h-224.17a61.24 61.24 0 00-61.24 61.24V726.72a61.24 61.24 0 0061.24 61.24h224.17a61.24 61.24 0 0061.24-61.24V327.37h3.862z"
          fill="#3f3d56"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M712.625 100.938h-29.262a21.728 21.728 0 01-20.116 29.933H534.822a21.728 21.728 0 01-20.117-29.933h-27.33a45.733 45.733 0 00-45.733 45.732v579.625a45.733 45.733 0 0045.733 45.732h225.25a45.733 45.733 0 0045.733-45.732V146.67a45.733 45.733 0 00-45.733-45.732z"
          fill="#fff"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M706.983 762.075H493.017c-24.585 0-44.586-17.852-44.586-39.796v-28.81c0-17.49 15.929-31.719 35.508-31.719H716.06c19.58 0 35.508 14.229 35.508 31.719v28.81c0 21.944-20.001 39.796-44.586 39.796z"
          fill="#e5e5e5"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M515.51 277.85h-47.728a3.229 3.229 0 01-3.225-3.225v-47.727a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225z"
          fill="#00739d"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M587.746 277.85h-47.728a3.229 3.229 0 01-3.225-3.225v-47.727a3.229 3.229 0 013.225-3.225h47.728a3.229 3.229 0 013.224 3.225v47.727a3.229 3.229 0 01-3.224 3.225zM659.982 277.85h-47.728a3.229 3.229 0 01-3.224-3.225v-47.727a3.229 3.229 0 013.224-3.225h47.728a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225zM732.218 277.85h-47.727a3.229 3.229 0 01-3.225-3.225v-47.727a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225zM515.339 366.21H467.61a3.229 3.229 0 01-3.225-3.224v-47.728a3.229 3.229 0 013.225-3.225h47.728a3.229 3.229 0 013.225 3.225v47.728a3.229 3.229 0 01-3.225 3.225zM587.575 366.21h-47.728a3.229 3.229 0 01-3.224-3.224v-47.728a3.229 3.229 0 013.224-3.225h47.728a3.229 3.229 0 013.225 3.225v47.728a3.229 3.229 0 01-3.225 3.225z"
          fill="#e5e5e5"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M659.811 366.21h-47.727a3.229 3.229 0 01-3.225-3.224v-47.728a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.728a3.229 3.229 0 01-3.225 3.225zM732.047 366.21H684.32a3.229 3.229 0 01-3.225-3.224v-47.728a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.728a3.229 3.229 0 01-3.225 3.225z"
          fill="#00739d"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M515.168 454.571H467.44a3.229 3.229 0 01-3.224-3.225V403.62a3.229 3.229 0 013.224-3.225h47.728a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225zM587.404 454.571h-47.727a3.229 3.229 0 01-3.225-3.225V403.62a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225zM659.64 454.571h-47.727a3.229 3.229 0 01-3.225-3.225V403.62a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225zM731.877 454.571h-47.728a3.229 3.229 0 01-3.225-3.225V403.62a3.229 3.229 0 013.225-3.225h47.728a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225zM514.997 542.931H467.27a3.229 3.229 0 01-3.225-3.224v-47.728a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.728a3.229 3.229 0 01-3.225 3.224zM587.234 542.931h-47.728a3.229 3.229 0 01-3.225-3.224v-47.728a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.728a3.229 3.229 0 01-3.224 3.224zM659.47 542.931h-47.728a3.229 3.229 0 01-3.225-3.224v-47.728a3.229 3.229 0 013.225-3.225h47.728a3.229 3.229 0 013.225 3.225v47.728a3.229 3.229 0 01-3.225 3.224zM731.706 542.931h-47.728a3.229 3.229 0 01-3.224-3.224v-47.728a3.229 3.229 0 013.224-3.225h47.728a3.229 3.229 0 013.225 3.225v47.728a3.229 3.229 0 01-3.225 3.224z"
          fill="#e5e5e5"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M515.51 740.291h-47.728a3.229 3.229 0 01-3.225-3.225V689.34a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225zM587.746 740.291h-47.728a3.229 3.229 0 01-3.225-3.225V689.34a3.229 3.229 0 013.225-3.225h47.728a3.229 3.229 0 013.224 3.225v47.727a3.229 3.229 0 01-3.224 3.225zM659.982 740.291h-47.728a3.229 3.229 0 01-3.224-3.225V689.34a3.229 3.229 0 013.224-3.225h47.728a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225zM732.218 740.291h-47.727a3.229 3.229 0 01-3.225-3.225V689.34a3.229 3.229 0 013.225-3.225h47.727a3.229 3.229 0 013.225 3.225v47.727a3.229 3.229 0 01-3.225 3.225z"
          fill="#fff"
          transform="translate(-184.5 -85.005)"
        />
        <circle
          cx="392.111"
          cy="548.221"
          fill="#e5e5e5"
          r="6.45"
        />
        <circle
          cx="415.329"
          cy="548.221"
          fill="#e5e5e5"
          r="6.45"
        />
        <circle
          cx="438.548"
          cy="548.221"
          fill="#e5e5e5"
          r="6.45"
        />
        <path
          d="M1014.5 789.005h-829a1 1 0 010-2h829a1 1 0 010 2z"
          fill="#cbcbcb"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M510.864 592.107a13.05 13.05 0 00-1.76-19.932l12.596-44.629-22.964 7.304-8.44 41.26a13.12 13.12 0 0020.568 15.997z"
          fill="#ffb7b7"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M517.394 564.752a5.97 5.97 0 01-1.245.105l-22.03-.441a5.84 5.84 0 01-5.66-6.693l22.348-150.991a18.512 18.512 0 0118.257-15.717 18.413 18.413 0 0117.83 22.826l-15.84 64.156-8.983 81.661a5.806 5.806 0 01-4.677 5.094z"
          fill="#3f3d56"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M342.105 718.105L354.365 718.105 360.198 670.817 342.103 670.818 342.105 718.105z"
          fill="#ffb7b7"
        />
        <path
          d="M523.478 799.108l24.144-.001h.001a15.387 15.387 0 0115.387 15.386v.5l-39.531.002z"
          fill="#2f2e41"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M394.382 714.123L406.565 715.489 417.632 669.146 399.65 667.13 394.382 714.123z"
          fill="#ffb7b7"
        />
        <path
          d="M576.22 794.802l23.994 2.69a15.387 15.387 0 0113.576 17.006l-.055.497-39.285-4.405zM515.099 539.96s-16.808 37.21-1.966 61.377l8.843 188.168h20.303l19.697-152 35 45-21.323 105.028 24.831 1.394 34.492-106.422s-26.498-117.348-35.249-120.174l-5.75-26.826z"
          fill="#2f2e41"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M507.618 572.627s-15.142 17.378 24.858 23.378 82-3 83-15-107.858-8.378-107.858-8.378z"
          fill="#cbcbcb"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M708.135 357.058a11.611 11.611 0 01-14.059 10.925l-15.12 21.805-15.491-5.957 21.784-30.492a11.674 11.674 0 0122.886 3.72z"
          fill="#ffb7b7"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M580.264 385.112l-21.378-9.671-27.39 1.958s-33.02 38.606-20.02 75.606l-8 129c51 14 74.183-12.258 111-1l-11.372-143.369z"
          fill="#3f3d56"
          transform="translate(-184.5 -85.005)"
        />
        <path
          d="M565.976 394.505l14.288-9.393s19.846-3 27.78 4.697c0 0 26.926 2.253 33.429 8.975l7.16 1.614 23.787-30.802 14.556 7.91-21 57-62.872 3.13z"
          fill="#3f3d56"
          transform="translate(-184.5 -85.005)"
        />
        <circle
          cx="554.72"
          cy="334.983"
          fill="#ffb7b7"
          r="33.745"
          transform="rotate(-28.663 296.109 653.56)"
        />
        <path
          d="M525.533 308.6c6.023-10.833 13.55-17.848 25.943-17.595s23.113-.236 28.689 10.834a18.207 18.207 0 002.143-8.979 10.283 10.283 0 014.54 10.345 7.51 7.51 0 003.707-6.165c4.136 1.376 5.212 7.205 3.119 11.028s-6.198 6.035-10.136 7.904-2.12 11.756-4.69 15.275c-1.796 2.457-2.616 5.467-3.826 8.259a.54.54 0 01-.245.255c-2.772 1.546-4.522 4.309-4.033 7.446a1.213 1.213 0 01-.04.71c-3.51 6.612-7.299 13.531-13.661 17.477a22.654 22.654 0 01-17.019 2.63c-6.503-1.554-12.126-5.866-16.18-11.184s-6.662-11.602-8.621-17.995c-1.739-5.672-3.003-11.736-1.624-17.505s6.047-11.112 11.962-11.558"
          fill="#2f2e41"
          transform="translate(-184.5 -85.005)"
        />
      </svg>
      <p
        class="emotion-15 emotion-16"
        color="#000"
        id="body1"
      >
        Diese Seite ist zurzeit noch nicht vollständig für mobile Geräte optimiert
      </p>
    </div>
  </div>
</body>
`;
